docs/how-to-guides/develop-a-flow/develop-evaluation-flow.md

```diff
@@ -149,17 +149,22 @@ In this case, the `grades` value would be `["Correct", "Incorrect"]` and the acc
 
 
 ## How to log metrics
-
+:::{admonition} Limitation
+You can only log metrics in an `aggregation node`, otherwise the metric will be ignored.
```

Comment:
do we have a link to full example?

Reply:
This section is part of the whole `develop evaluation flow`, the full example is at the top of this page